### PR TITLE
fix: use lodash version of unsupported Array.{fill, find}

### LINKED ISSUE #1115: IE 11 doesn't support Array.find()
:wave: Hi team

IE 11 is throwing an error when I try to test on my local development server because of the use of Array.find(). I suspect it's the use here: https://github.com/algolia/react-instantsearch/blob/1a164d5536c82253b96619f4d2ba8d6964ae0992/packages/react-instantsearch/src/core/createConnector.js#L85-L93

which is only used when `process.env.NODE_ENV === 'development'`.

For now I'm just polyfilling it myself, but supporting the same browsers in production and development seems like the best solution. What do you say? 🙂

### PR DESCRIPTION
**Pull request by samouss**
**Summary**

Close #1115

### CODE REVIEW

File: packages/react-instantsearch/src/components/RatingMenu.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import { find, fill } from 'lodash';
```
Comment (Haroenv) [inline packages/react-instantsearch/src/components/RatingMenu.js]: probably best to depend on /find and /fill ?

Comment (samouss) [reply packages/react-instantsearch/src/components/RatingMenu.js]: The `babel-rollup-plugin` handle that for us.

Comment (Haroenv) [reply packages/react-instantsearch/src/components/RatingMenu.js]: didn't know we used it 🙌

**Comment by vvo** [comment]
In the future if we want to get rid of lodash, maybe we can use the automatic polyfill features of babel? Any of you have experience using https://babeljs.io/docs/plugins/transform-runtime?

**Comment by Haroenv** [comment]
Would be better to use core-js directly rather than babel, I think that seems to be more of the recommendation. I fear that core-js is actually bigger than loads usually too

**Comment by samouss** [comment]
`babel-runtime` use `core-js` under the hood. I think it's the same at the end.

**Pull request merged**